platform/11-tekton-chains.sh

```diff
@@ -13,3 +13,4 @@ kubectl patch \
       configmap chains-config \
       -n tekton-chains \
       --patch-file "$GIT_ROOT"/platform/components/tekton/chains/patch_config_oci.yaml
+
```

Comment:
This seems to be an unintentional whitespace change

Reply:
I believe this was semi-intentional to prevent the "No new line at end of file" error. But to be fair it is added automatically by editor config.